### ISSUE TITLE
Full text search for SQL server?

### ISSUE THREAD

**Issue by martoiu** [sqlkata/querybuilder#567]
It would be great if the 'Where' could also format user input for full text search in SQL server.

**Comment by ahmad-moussawi** [comment]
@martoiu do you have any proposal or suggested syntax for that?

**Comment by martoiu** [comment]
> @martoiu do you have any proposal or suggested syntax for that?

No. There are more possibilities. One could start with the simplest.  'Contains' for instance.
